aws/main.tf

```diff
@@ -0,0 +1,60 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+locals {
+  yaml_secrets= yamldecode(file("../secrets.yaml"))
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_ssm_parameter" "github_pat" {
+  name  = "/github/pat"
+  type  = "String"
+  value = local.yaml_secrets.samples.github_pats[0]
+}
```

Comment:
Should we use a for loop (or count) to create multiple parameters?

Reply:
I think it is better to have all the secrets as there aren't a lot

Reply:
Works for me!